packages/core/src/common/cancellation.ts

```diff
@@ -21,7 +21,8 @@ export interface CancellationToken {
 }
 
 const shortcutEvent: Event<void> = Object.freeze(function (callback: any, contex?: any): any {
```

Comment:
contex vs context - there is a typo in the parameter name

Reply:
that's the thing :) I was struggling to understand why a variable is unknown :)